### PR TITLE
[CodingStyle] Handle VarConstantCommentRector+NewlineAfterStatementRector

### PR DESCRIPTION
**Pull request by samsonasik**
ref https://github.com/rectorphp/rector-src/pull/2623#discussion_r912522275

### CODE REVIEW

**Comment by samsonasik** [comment]
Fixed 🎉 , the `NodesToAddCollector` is needed due to add `Nop` Stmt for this use case.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/CodingStyle/Rector/Stmt/NewlineAfterStatementRector.php [hunk 1]
```diff
@@ -146,7 +150,8 @@ public function refactor(Node $node): ?array
             }
         }
 
-        return [$node, new Nop()];
+        $this->nodesToAddCollector->addNodeAfterNode(new Nop(), $node);
+        return $node;
```
Comment (TomasVotruba) [inline rules/CodingStyle/Rector/Stmt/NewlineAfterStatementRector.php]: The `nodesToAddCollector` is about to be removed unless on stmt :slightly_smiling_face: 
The array of nodes must be returned here.

Comment (samsonasik) [reply rules/CodingStyle/Rector/Stmt/NewlineAfterStatementRector.php]: the issue possibly on auto mirroring comment on return array of nodes at AbstractRector

https://github.com/rectorphp/rector-src/blob/a70d0e5518260479d055a0d5a90e28e43cad58f7/src/Rector/AbstractRector.php#L237-L238

that, mirroring `Nop` for new Node cause issue, and then printed as skipped.

https://github.com/rectorphp/rector-src/blob/a70d0e5518260479d055a0d5a90e28e43cad58f7/src/PhpParser/Printer/BetterStandardPrinter.php#L533

Comment (samsonasik) [reply rules/CodingStyle/Rector/Stmt/NewlineAfterStatementRector.php]: @TomasVotruba ok, fixed with compare Node before mirror comment in `AbstractRector` https://github.com/rectorphp/rector-src/pull/2629/commits/42f5287815ec97f1dc14e9f7602c807cc7b0a294 🎉

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by samsonasik** [comment]
rebased.

**Comment by TomasVotruba** [comment]
Thanks :+1:

**Pull request merged**